emission/individual_tests/TestNominatim.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import *
+import unittest
+import unittest.mock as mock
+# import overpy
+import os
+from emission.core.wrapper.trip_old import Coordinate
+import requests
+import emission.core.wrapper.entry as ecwe
+import emission.core.wrapper.wrapperbase as ecww
+import emission.net.ext_service.geocoder.nominatim as eco
+import emission.analysis.intake.cleaning.clean_and_resample as clean
+
+#temporarily sets NOMINATIM_QUERY_URL to the environment variable for testing.
+NOMINATIM_QUERY_URL_env = os.environ.get("NOMINATIM_QUERY_URL")
+
+NOMINATIM_QUERY_URL = (
+    NOMINATIM_QUERY_URL_env if NOMINATIM_QUERY_URL_env is not None 
+    else eco.NOMINATIM_QUERY_URL
+)
+
+GFBK = os.environ.get("GFBK_KEY")
+
+GEOFABRIK_QUERY_URL = (
+    "https://geocoding.geofabrik.de/{}".format(GFBK) if GFBK is not None
+    else print("No key available")
+)
+
+#Creates a fake place in Rhode Island to use for testing.
+fake_id = "rhodeislander"
+key = "segmentation/raw_place"
+write_ts = 1694344333
+data = {'source': 'FakeTripGenerator','location': {'type': 'Point', 'coordinates': [-71.4128343, 41.8239891]}}
+fake_place = ecwe.Entry.create_fake_entry(fake_id, key, data, write_ts)
+
+class NominatimTest(unittest.TestCase):
+    maxDiff = None
+
+    #Basic query to check that nominatim and geofabrik are calling the same area.
```

Comment:
the same area? you mean "return the same results"?

Reply:
I meant it as in the same location, since the exact results vary between geofabrik, nominatim, and the container

Reply:
but the check is not whether they are calling the same area or not, it is whether they are returning the same results or not

Reply:
Changed this comment